draft-ietf-quic-transport.md

```diff
@@ -1875,46 +1874,20 @@ Congestion Experienced (CE) codepoint set by a network device that is
 experiencing congestion.
 
 On receiving a packet with an ECT or CE codepoint, an endpoint that can access
-the IP ECN codepoints increases the corresponding ECT(0), ECT(1), or CE count,
-and includes these counters in subsequent (see {{processing-and-ack}}) ACK_ECN
-frames (see {{frame-ack-ecn}}).
+the IP ECN codepoints records the markings for that packet.  This information is
+reported in ACK frames (see {{processing-and-ack}}).  Packets bearing ECT(1) are
+reported as though they were not marked.
 
```

Comment:
I'm not sure if this is the best choice. I think it would be more future-proof to have the ACK block type (2) as meaning either ECT(0) or ECT(1) marked because that makes it possible to switch over to a mechanism that uses ECT(1) as default marking without changing the feedback (which is the current idea of these the codepoint - basically you mark all you packets either ECT(0) or all them ECT(1) but you don't use both in the same connection as ECN Nonce did). 

However, no providing feedback on one of the marking, also means testing is less easy. If you could use both of the codepoints at least for testing you can detect if there is a box overwriting the codepoint to either ECT(0) or ECT(1). However, I guess you can find another mechanism to get this kind of feedback if needed, e.g. tell the other endpoint what the default ECT is that you are planning to use in the TP and then have a separate feedback frame that tells you if the other unexpected ECT was observed.

The other option is of course to use a third bit for one of the cases, e.g. if the ACK block type is (2) you need to check the next bit that indicates you if this was a ECT(0) or ECT(1). Or you could use the third bit to indicate either a gap/loss or CE mark as both blocks should usually be rather small and both indicate congestion.